modules/nf-core/parabricks/indexgvcf/main.nf

```diff
@@ -1,16 +1,17 @@
 process PARABRICKS_INDEXGVCF {
     tag "$meta.id"
     label 'process_high'
+    label 'process_gpu'
 
-    container "nvcr.io/nvidia/clara/clara-parabricks:4.3.0-1"
+    container "nvcr.io/nvidia/clara/clara-parabricks:4.4.0-1"
 
     input:
-    tuple val(meta), path(gvcf, stageAs:'')
+    tuple val(meta), path(gvcf)
 
     output:
     // This tool outputs g.vcf.idx if input is uncompressed, g.vcf.gz.tbi if input is compressed
-    tuple val(meta), path("*.g.vcf*") , emit: gvcf_index
-    path "versions.yml"               , emit: versions
+    tuple val(meta), path("*") , emit: gvcf_index
```

Comment:
Literally anything seems a bit extreme, wouldn't that also pick up the `versions.yml` too?

Reply:
I am confused - what do you mean by picking up the versions too? And why remove the versions output???

Reply:
You are specifying literally any output file generated by the entire module, is there not a specific suffix you can give your pattern:

```suggestion
path("*.{idx,tbi}")
```

With just `*` your `gvcf_index` may also include the `versions.yml` file, I believe?